client/client.go

```diff
@@ -273,6 +283,22 @@ func (sc *ServerConnection) Send(tags map[string]string, prefix string, command
 	return nil
 }
 
+// dispatchCommand dispatches an event based on simple commands (e.g !help)
+func (sc *ServerConnection) dispatchCommand(info eventmgr.InfoMap) {
+	params := strings.Split(info["params"].([]string)[1], " ")
```

Comment:
Feels like it'd make sense to use [`strings.Fields`](https://golang.org/pkg/strings/#Fields) here to split it up instead of strings.Split, just to ensure we catch the case of multiple spaces between params properly

Reply:
Oh neat. I didn't realise that was a thing